src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -108,6 +109,7 @@ export async function poll(
   );
 
   const breaches = await getAllBreachesFromDb();
+  const subPlatFeatureFlag = await getFeatureFlagData("SubPlat3");
```

Comment:
Any reason not to enable `getEnabledFeatureFlags`? We already changed the allowlist logic once, so there is some risk to not re-using the logic, and potentially future contributors will modify `sendEmail` and not realise that not all feature flags show up in that parameter?

Reply:
We can also use `getEnabledFeatureFlags` here. I thought there might have been a reason to be more precise in getting the feature flag as it has been also used in [cronjobs/churnDiscount](https://github.com/mozilla/blurts-server/blob/main/src/scripts/cronjobs/churnDiscount.tsx#L34). Or was this pre-changing the allowlist logic?

Reply:
Ah yes, there is indeed a reason, I should've added a comment - that way, I only needed a single query for all recipients.

Hmm... I imagine that might apply here too. It would probably be better to split up `getEnabledFeatureFlags` into a DB function that fetches all the feature flag data, and then another one that filters it down to a single user, but that's out of scope for this PR.

Then maybe at least put a comment above the `enabledFeatureFlags` to call out that this is a trap that future contributors might run into.

Reply:
Yes, I was just about to update to using `getEnabledFeatureFlags` and this would need a call for each subscriber instead of one for the cronjob.